components/combobox/stories/combobox.stories.js

```diff
@@ -165,6 +171,20 @@ QuietGroup.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+/**
+ * Combo boxes have a read-only option for when content in the disabled state still needs to be shown. This allows for content to be copied, but not interacted with or changed. A combo box does not have a read-only option if no selection has been made.
+*/
+
+export const ReadOnly = Template.bind({});
```

Comment:
Would you add `isReadOnly` to the `stateData` in the test file, please? It'd probably be a good idea to capture the read-only state, especially for the open test case.

```
{
	testHeading: "Read-only",
	isReadOnly: true,
},
```

<img width="1108" alt="Screenshot 2024-10-15 at 2 00 55 PM" src="https://github.com/user-attachments/assets/a8b48829-1598-40c0-a748-c18b93254240">